exercises/elm-1-a/tests/Tests.elm

```diff
@@ -0,0 +1,27 @@
+module Tests exposing (tests)
+
+import Expect
+import AggregateScorers exposing (..)
+import Test exposing (..)
+
+
+tests : Test
+tests =
+    describe "AggregateScorers"
+        [ test "doesn't change a single scorer" <|
+            \() ->
+                aggregateScorers [ "Anna" ]
+                |> Expect.equal [ "Anna" ]
+        , test "doesn't change multiple unique scorers" <|
+            \() ->
+                aggregateScorers [ "Anna", "Lisa" ]
+                |> Expect.equal [ "Anna", "Lisa" ]  
+        , test "converts duplicates in to goals" <|
```

Comment:
Maybe add a test case for the example mentioned before, where there are multiple scorers and one of them has scored more than once?